### PR TITLE
Sa 2600 di logs

### PR DESCRIPTION
**Pull request by kmaranionjc**
## Issues
* [SA-2600](https://jumpcloud.atlassian.net/browse/<jira-id>) - SA-2600 DI app logs for current event data

## What does this solve?
Log gathering of service, timestamps of current DI event when app is running then create and log Powershell script to run manually incase of errors or timeouts during the event

## Is there anything particularly tricky?
Validating AWS DI json data to the powershell script DI json data. The outputs of the powershell script and DI App are not the in same order 

## How should this be tested?
*** AWS ***
1. Deploy
2.  Go to the Lambda function
3. Click on the Monitor tab
4. View logs in CloudWatch
5. Go to the most recent Log Stream
           - Click 'view as text' to make the logs easily readable

*** GCP ***
1. Deploy
2. Go to your function in Cloud Functions
3.  Click on the Actions button on the Actions column then view logs
4. Severity to default and sort timestamp to newest first
## Screenshots
image

image

### CODE REVIEW

**Review by gweinjc — APPROVED** [review]
Looks good to me!

File: AWS/DirectoryInsights/get-jcdirectoryinsights.py [hunk 1]
```diff
@@ -46,6 +48,13 @@ def jc_directoryinsights(event, context):
             raise Exception(f"Error - Service List contains 'all' and additional services : {serviceList}")
     finalData = []
 
+    if len(serviceList) > 1:
```
Comment (jworkmanjc) [inline AWS/DirectoryInsights/get-jcdirectoryinsights.py:51]: I'm unsure about this, If the two strings generated are the same why would we break this out into a if statement?

File: AWS/DirectoryInsights/get-jcdirectoryinsights.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import requests, datetime, json, boto3, os, gzip
+import requests, datetime, json, boto3, os, gzip, logging
```
Comment (jworkmanjc) [inline AWS/DirectoryInsights/get-jcdirectoryinsights.py:1]: In addition to installing boto3 do we need to deploy with the logging libraries ex:

`~/jc-directoryinsights$ pip3 install logging -t .`

Or is the logging library something that's included by default?

Comment (kmaranionjc) [reply AWS/DirectoryInsights/get-jcdirectoryinsights.py:1]: Included, no need to do anything.

File: AWS/DirectoryInsights/get-jcdirectoryinsights.py (hunk 1 above)
Comment (kmaranionjc) [reply AWS/DirectoryInsights/get-jcdirectoryinsights.py:51]: If there are >1 services, it creates multiple log for each service

Comment (jworkmanjc) [reply AWS/DirectoryInsights/get-jcdirectoryinsights.py:51]: We reviewed and will produce multiple logs for multiple services

**Pull request merged**